projects/plugins/jetpack/extensions/plugins/publicize/components/panel/index.js

```diff
@@ -48,30 +45,25 @@ const PublicizePanel = ( { prePublish } ) => {
 		[ hasEnabledConnections, refresh ]
 	);
 
-	// Disable the panel when no proper site plan is available.
-	const PanelRowWithDisabled = isPublicizeDisabledBySitePlan ? Disabled : PanelRow;
-
 	// Panel wrapper.
 	const PanelWrapper = prePublish ? Fragment : PanelBody;
 	const wrapperProps = prePublish
 		? {}
 		: {
 				title: __( 'Share this post', 'jetpack' ),
-				className: isPublicizeDisabledBySitePlan ? 'jetpack-publicize-disabled' : '',
+				className: '',
```

Comment:
Can we just get rid of the `className` prop completely?

Reply:
Done!